app/app.controller.js

```diff
@@ -12,4 +13,12 @@ function getAllTopics(req, res, next) {
     });
 }
 
-module.exports = getAllTopics;
+function getAllEndpoint(req, res, next){
+  selectAllEndpoint()
+  .then((endpoints) =>{
+    res.status(200).send({endpoints})
+  })
```

Comment:
We need to make sure to also invoke next(err), in order to pass on any potential error to app.js